addon/mixins/legend.js

```diff
@@ -88,6 +88,16 @@ export default Ember.Mixin.create({
     return this.get('numLegendRows') * this.get('legendItemHeight');
   }),
 
+  // Bottom margin is equal to the total amount of space the legend needs,
+  _marginBottom: Ember.computed('legendHeight', 'hasLegend', 'marginTop', function() {
+    // If the legend is enabled then we need some extra breathing room
+    return this.get('hasLegend') ? this.get('legendHeight') : this.get('marginBottom');
+  }),
+
+  marginBottom: Ember.computed('_marginBottom', 'minimumTopBottomMargin', function() {
+    return Math.max(this.get('_marginBottom'), this.get('minimumTopBottomMargin'));
+  }),
+
```

Comment:
The legend mixin isn't included when we need to set the margin explicitly.  We end up setting the margin on the bottom of the chart directly.  This code overrides that code.  It's not obvious, but a lot of properties are overriden through the mixin and extends structure.